### PR TITLE
Adding jupyter{book} for Demo Data Center

### PR DESCRIPTION
**Pull request by lohithmunakala**
There was a notebook issue and that has been resolved. This is with respect to the PR https://github.com/ioos/notebooks_demos/pull/408 and https://github.com/ioos/notebooks_demos/pull/394 . The updated branch without conflicts is mentioned here. 

The view is the same as shown in #408. 

@ocefpaf please let me know if this is fine.

### CODE REVIEW

**Comment by lohithmunakala** [comment]
@MathewBiddle @ocefpaf are these fails common because the others PR's seem to have the same issue?

**Comment by ocefpaf** [comment]
The PR is massive, 615 files changed! That is quite hard to deal with.

One way to reduce would be to reset the changes in the notebooks. Changes like this one: https://github.com/ioos/notebooks_demos/pull/409/files#diff-7c024449328887c11902b4b4b457ee5a50f32dc94de8fcc7422bb9c9810f9ca0

Can you do that? Also, please confirm upstream if we need to commit the `_build` directory. That would also reduce the noise level on the PR. Or maybe we can commit them in another PR?

**Comment by lohithmunakala** [comment]
Yes I will do that. Will look at reducing the redundancy of the PR

**Comment by lohithmunakala** [comment]
I have removed all the build files to keep the initial PR clean. Please check and let me know @ocefpaf.

File: jupyterbook/_config.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+title: The U.S. Integrated Ocean Observing System (IOOS)
+author: IOOS 
+copyright: "2021"
+logo: images/IOOS-logo_little.png
+
+#only build files that are there in the table of contents. else defaults to building all files 
+only_build_toc_files: true
+
+launch_buttons:
+  notebook_interface: "classic" # The interface interactive links will activate ["classic", "jupyterlab"]
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:10]: @MathewBiddle I have a preference for the classic style notebook but I am old. What is your opinion on this?

File: jupyterbook/_config.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  binderhub_url: "https://mybinder.org"
+  colab_url: "https://colab.research.google.com"
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:12]: @lohithmunakala can we remove this? I think that having many ways to run notebooks can be confusing. Colab is great for ML and/or when you need a GPU but most of the notebooks we have run OK on regular binder.

File: jupyterbook/_config.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  thebe: true
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:13]: Does this work with our multiple kernels? It is an awesome feature but I never tested it with conda.

File: jupyterbook/_config.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+
+#######################################################################################
+# Execution settings
+execute:
+  execute_notebooks           : cache
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:19]: What exactly does this mean? The JB executes once and re-execute if there are changes?

File: jupyterbook/_config.yml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+#######################################################################################
+# HTML-specific settings
+html:
+  home_page_in_navbar         : false
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:24]: @lohithmunakala do you mean sending a screen shot with and without this so we can choose? I believe we may want that.

File: jupyterbook/_config.yml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+# Interact link settings #
+notebook_interface            : "notebook"
+
+#######################################################################################
+# Launch button settings
+#repository:
+#  url                         : https://github.com/
+
+#  use_repository_button     : true  # Whether to add a link to your repository button
+#  use_issues_button         : true  # Whether to add an "open an issue" button
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:37]: We need the open issue. That is how we interact with users to request new notebooks.

File: jupyterbook/_config.yml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+latex:
+  latex_engine                : "xelatex"
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:39]: Probably not necessary but I guess it only runs if you want a pdf version, right?

File: jupyterbook/_toc.yml [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+      sections:
+          - file: content/Code Gallery/data_access/2016-10-12-fetching_data.ipynb
+          - file: content/Code Gallery/data_access/2017-05-14-running_compliance_checker.ipynb
+          - file: content/Code Gallery/data_access/2016-12-20-searching_glider_deployments.ipynb
+          - file: content/Code Gallery/data_access/2017-01-18-siphon-explore-thredds.ipynb
+          - file: content/Code Gallery/data_access/2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb
+          - file: content/Code Gallery/data_access/2017-06-12-NCEI_RA_archive_history.ipynb
+          - file: content/Code Gallery/data_access/2017-07-25-HFRadar_currents.ipynb
+          - file: content/Code Gallery/data_access/2017-12-15-finding_HFRadar_currents.ipynb
+          - file: content/Code Gallery/data_access/2018-03-01-erddapy.ipynb
```
Comment (ocefpaf) [inline jupyterbook/_toc.yml:17]: This is tricky :-/

Currently our notebooks are listed by date and we don't need to list them anywhere to publish them. If we cannot automate the creation of this TOC we should document it. @lohithmunakala can you create a wikipage with the process that one would take to "add a new notebook?"

File: jupyterbook/_toc.yml [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+    - file: content/Code Gallery/R_Matlab_Notebooks/R_Matlab_Notebooks.md
+      sections:
+        - file: content/Code Gallery/R_Matlab_Notebooks/2017-01-23-R-notebook.ipynb
+        - file: content/Code Gallery/R_Matlab_Notebooks/2017-03-30-octave_notebook_example.ipynb
+        - file: content/Code Gallery/R_Matlab_Notebooks/2017-08-01-xtractoR.ipynb
+        - file: content/Code Gallery/R_Matlab_Notebooks/2018-02-20-obis.ipynb
+        - file: content/Code Gallery/R_Matlab_Notebooks/2017-11-30-rerddap.ipynb
+
+
+- part: Video Tutorials
```
Comment (ocefpaf) [inline jupyterbook/_toc.yml:50]: @MathewBiddle I wonder if we should still have the videos in the JB. They are helpful but seems a bit odd and kind of mixed[content-out-if-context.

File: jupyterbook/_toc.yml [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+      title: Python Demonstration Videos
+
+- part: Other Resources   
+  chapters:
+    - file: content/Other Resources/ioos_installation_conda.md
+      title: Installing the IOOS Conda Environment
+    - url: https://help.aodn.org.au/aodn-data-tools/user-code-library/
+      title: Opening netCDF files - hints from AODN
+    - url: http://unidata.github.io/notebook-gallery/
+      title: Unidata Ju[yter Notebook Gallery
```
Comment (ocefpaf) [inline jupyterbook/_toc.yml:70]: Typo in Jupyter.

File: jupyterbook/environment.yml [hunk 11]
```diff
@@ -0,0 +1,82 @@
+name: IOOS
```
Comment (ocefpaf) [inline jupyterbook/environment.yml:1]: Can we read from the .binder directory instead? To avoid duplication of this file.

File: jupyterbook/content/Code Gallery/data_analysis_and_visualization/data_analysis_and_visualization.md [hunk 12]
```diff
@@ -0,0 +1,3 @@
+# Data Analysis And Visualization 
+
+This chapter will consist of notebooks which represent data analysis.
```
Comment (ocefpaf) [inline jupyterbook/content/Code Gallery/data_analysis_and_visualization/data_analysis_and_visualization.md:3]: Question. This markdown file is the "chapter" and the notebooks inside the folder with it are the contents. Do we need to match the folder and markdown file name? Or is that set elsewhere?

File: jupyterbook/content/Code Gallery/R_Matlab_Notebooks/2017-08-01-xtractoR.ipynb [hunk 13]
```diff
@@ -0,0 +1,259 @@
+{
```
Comment (ocefpaf) [inline jupyterbook/content/Code Gallery/R_Matlab_Notebooks/2017-08-01-xtractoR.ipynb:1]: It is hard to follow if these diffs are due to directory move or if they really have differences. But that is fine for now. We cannot escape from reviewing the final HTML at some point.

**Comment by ocefpaf** [comment]
> I have removed all the build files to keep the initial PR clean. Please check and let me know @ocefpaf.

This is great. We are down to 53 files :-) much easier to review.

Please take a look at my comments at your convenience.

File: jupyterbook/_config.yml (hunk 2 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: Ok, will remove Colab for now. I Will keep a blank string there.

File: jupyterbook/_config.yml (hunk 3 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: Yes, this does work with multiple kernels. I am actually uploading another `config.yaml` file with comments so that it becomes easier to understand for anyone else using it.

File: jupyterbook/_config.yml (hunk 4 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: Yes. It caches all the notebook data into a folder. If there is any new change in the cells of the code/notebook, it re-builds the Jupyter{Book}.

File: jupyterbook/_config.yml (hunk 5 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: #### Below is the web-app with `home_page_in_navbar : true` 
Screenshot from 2021-06-21 00-51-28

#### Below is the web-app with `home_page_in_navbar : false` 
Screenshot from 2021-06-21 00-52-24

File: jupyterbook/_config.yml (hunk 6 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: Okay. Have turned this on.

File: jupyterbook/_config.yml (hunk 7 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: There is a download option that is available for each of the notebooks. The `latex_engine` option is for the same.

File: jupyterbook/_toc.yml (hunk 8 above)
Comment (lohithmunakala) [reply jupyterbook/_toc.yml:17]: I am thinking of writing a .sh file that will automate this process. But for now, sure, I will write a wikipage for manually changing the `toc.yaml` file.

File: jupyterbook/environment.yml (hunk 11 above)
Comment (lohithmunakala) [reply jupyterbook/environment.yml:1]: Sure. We don't actually need the env.yml file for building the web app. Will remove it.

File: jupyterbook/content/Code Gallery/data_analysis_and_visualization/data_analysis_and_visualization.md (hunk 12 above)
Comment (lohithmunakala) [reply jupyterbook/content/Code Gallery/data_analysis_and_visualization/data_analysis_and_visualization.md:3]: No, we **don't need to** keep them the same. I kept them like that to avoid confusion. I will change it if you would want me to.

**Comment by lohithmunakala** [comment]
> > I have removed all the build files to keep the initial PR clean. Please check and let me know @ocefpaf.
> 
> This is great. We are down to 53 files :-) much easier to review.
> 
> Please take a look at my comments at your convenience.

I have made quite some changes to the `_config.yml` file. Included the above changes and made it easier to understand and included all the default values too (that were previously not included).

File: jupyterbook/_config.yml (hunk 1 above)
Comment (MathewBiddle) [reply jupyterbook/_config.yml]: @ocefpaf @lohithmunakala, can you point me to a jupyterlab example?

File: jupyterbook/_config.yml (hunk 2 above)
Comment (MathewBiddle) [reply jupyterbook/_config.yml]: @ocefpaf, the one thing I do like about google colab is it's easy to spin up and play with a notebook (I've been using it alot with some of the more off the shelf notebooks). With binder a user has to wait for the environment to build, which can take quite a while depending on the env. How much more overhead is it to enable this?

File: jupyterbook/_config.yml (hunk 6 above)
Comment (MathewBiddle) [reply jupyterbook/_config.yml]: Are there other features here, like star the notebook or give a +1 or something like that to get more feedback from users?

File: jupyterbook/_toc.yml (hunk 8 above)
Comment (MathewBiddle) [reply jupyterbook/_toc.yml:17]: before you get too far, I started this [wiki page](https://github.com/ioos/notebooks_demos/wiki) to document some of that information. It probably needs to be expanded upon. 

@ocefpaf what was the thinking behind adding dates in the titles of the notebooks? Is this something we might want to reevaluate in this new paradigm?

File: jupyterbook/_toc.yml (hunk 9 above)
Comment (MathewBiddle) [reply jupyterbook/_toc.yml:50]: I don't know just yet. I really like [this resource](https://www.esipfed.org/data-help-desk-at-egu-2021) that ESIP put together for the data help desk at EGU-2021. If possible, I would like to replicate it with our videos. I'm just not sure how to just yet...

File: jupyterbook/_config.yml (hunk 1 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: This feature is specifically when we have a JupyterHub to run these notebooks just like running them on Colab/Binder. Other than that there isn't any change in the actual appearance of the Jupyter{Book}.

However, I've added examples of classic and JupyterLab below, in case we have JupyterHub, we can use either of these to run the notebooks.
#### Below is a JupyterLab Notebook
Screenshot from 2021-06-22 00-25-32
#### Below is a Classic Jupyter Notebook
Screenshot from 2021-06-22 00-25-39

File: jupyterbook/_config.yml (hunk 2 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: @MathewBiddle I don't think the environment would be installed on a Colab notebook while compared to binder.

File: jupyterbook/_config.yml (hunk 6 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: There isn't any option of upvoting the notebook for now but might get it in the future builds of Jupyter{Book}.

File: jupyterbook/_toc.yml (hunk 8 above)
Comment (lohithmunakala) [reply jupyterbook/_toc.yml:17]: Sure will add changes to the wiki file.

File: jupyterbook/_config.yml [hunk 14]
```diff
@@ -46,7 +46,7 @@ html:
   use_issues_button         : true  # Whether to add an "open an issue" button
   extra_navbar              : Powered by <a href="https://jupyterbook.org">Jupyter Book</a>  # Will be displayed underneath the left navbar.
   extra_footer              : ""  # Will be displayed underneath the footer.
-  google_analytics_id       : ""  # A GA id that can be used to track book views.
+  google_analytics_id       : "G-W7S16LPGS7"  # A GA id that can be used to track book views.
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:49]: :+1:

File: jupyterbook/_config.yml [hunk 15]
```diff
@@ -0,0 +1,81 @@
...
+    # - replacements
+    # - smartquotes
+    - substitution
+
+  myst_url_schemes          : [mailto, http, https]  # URI schemes that will be recognised as external URLs in Markdown links
+
+#######################################################################################
+# HTML-specific settings
+html:
+  favicon                   : ""  # A path to a favicon image
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:43]: I believe we have that in the current site. Can you check it?

File: jupyterbook/_config.yml [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+  use_edit_page_button      : false  # Whether to add an "edit this page" button to pages. If `true`, repository information in repository: must be filled in
+  use_repository_button     : false  # Whether to add a link to your repository button
+  use_issues_button         : true  # Whether to add an "open an issue" button
+  extra_navbar              : Powered by <a href="https://jupyterbook.org">Jupyter Book</a>  # Will be displayed underneath the left navbar.
+  extra_footer              : ""  # Will be displayed underneath the footer.
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:48]: @MathewBiddle do we want this?

File: jupyterbook/_config.yml [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+  use_jupyterbook_latex     : true # use jupyterbook-latex for pdf builds as default
+
+#######################################################################################
+# Launch button settings
+launch_buttons:
+  notebook_interface        : classic  # The interface interactive links will activate ["classic", "jupyterlab"]
+  binderhub_url             : https://binder.pangeo.io  # The URL of the BinderHub (e.g., https://mybinder.org)
+  jupyterhub_url            : ""  # The URL of the JupyterHub (e.g., https://datahub.berkeley.edu)
+  thebe                     : false  # Add a thebe button to pages (requires the repository to run on Binder)
+  colab_url                 : " " # The URL of Google Colab (https://colab.research.google.com)
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:69]: @lohithmunakala I spoke with @MathewBiddle and we do want this but we need a page documentation that the user may need to `pip install` extra modules.

File: jupyterbook/_toc.yml [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+          - file: content/Code Gallery/data_access/2017-05-14-running_compliance_checker.ipynb
+          - file: content/Code Gallery/data_access/2016-12-20-searching_glider_deployments.ipynb
+          - file: content/Code Gallery/data_access/2017-01-18-siphon-explore-thredds.ipynb
+          - file: content/Code Gallery/data_access/2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb
+          - file: content/Code Gallery/data_access/2017-06-12-NCEI_RA_archive_history.ipynb
+          - file: content/Code Gallery/data_access/2017-07-25-HFRadar_currents.ipynb
+          - file: content/Code Gallery/data_access/2017-12-15-finding_HFRadar_currents.ipynb
+          - file: content/Code Gallery/data_access/2018-03-01-erddapy.ipynb
+
+    - file: content/Code Gallery/data_analysis_and_visualization/data_analysis_and_visualization.md
```
Comment (ocefpaf) [inline jupyterbook/_toc.yml:19]: Can we auto-generate this file based on the directory structure?

File: jupyterbook/_config.yml (hunk 15 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml:43]: It is the IOOS Logo. Will add and commit changes.

File: jupyterbook/_config.yml (hunk 5 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml]: @ocefpaf, any update regarding this?

File: jupyterbook/_config.yml (hunk 1 above)
Comment (MathewBiddle) [reply jupyterbook/_config.yml]: I kind of prefer the JupyterLab view. Reminds me of how I have Pycharm configured :P But, I don't have strong attachments to either.

File: jupyterbook/_config.yml (hunk 5 above)
Comment (MathewBiddle) [reply jupyterbook/_config.yml]: So, this is simply adding the `IOOS Data Demo Center` link in the navigation on the left? If we keep that in the navbar, we should change the text so it's clear what that is linking to. I'd recommend "Home" or "IOOS Data Demo Center Home".

Comment (lohithmunakala) [reply jupyterbook/_config.yml]: The text will be the name of the homepage. As the name of the homepage is `IOOS Data Demo Center,` it shows up like that. If we rename the homepage to `IOOS Data Demo Center Home` , the text would get renamed automatically. 

We can still go to the homepage by clicking on the IOOS Logo/Name.

Comment (MathewBiddle) [reply jupyterbook/_config.yml]: If clicking on the IOOS logo gets you there, I'd say this is duplicative and can be removed.

File: jupyterbook/_config.yml (hunk 17 above)
Comment (MathewBiddle) [reply jupyterbook/_config.yml:69]: Probably something on the [intro page](https://github.com/lohithmunakala/notebooks_demos/blob/jupyterbook3/jupyterbook/content/intro.md) that says

In order to use the google colab functionality for these notebooks, some extra modules might need to be installed via `pip install`. See this information on [importing a library that is not in Colaboratory](https://colab.research.google.com/notebooks/snippets/importing_libraries.ipynb).

File: jupyterbook/_config.yml (hunk 16 above)
Comment (MathewBiddle) [reply jupyterbook/_config.yml:48]: Yes, if possible:

- [ ] Add a standardized, centered IOOS header/masthead logo that links back to ioos.us
- [ ] Make the IOOS image on the upper left point to ioos.us (we'll need to add the home link back in to the menu probably)
- [ ] footer: standardize to [ioos.us](https://ioos.us/) design, add link to ioos.noaa.gov, contact info etc. Should have consistent look and feel if possible and match the content at the main ioos.us page

File: jupyterbook/_toc.yml [hunk 19]
```diff
@@ -1,68 +1,64 @@
...
-          - file: content/Code Gallery/data_access/2016-10-12-fetching_data.ipynb
-          - file: content/Code Gallery/data_access/2017-05-14-running_compliance_checker.ipynb
-          - file: content/Code Gallery/data_access/2016-12-20-searching_glider_deployments.ipynb
-          - file: content/Code Gallery/data_access/2017-01-18-siphon-explore-thredds.ipynb
-          - file: content/Code Gallery/data_access/2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb
-          - file: content/Code Gallery/data_access/2017-06-12-NCEI_RA_archive_history.ipynb
-          - file: content/Code Gallery/data_access/2017-07-25-HFRadar_currents.ipynb
-          - file: content/Code Gallery/data_access/2017-12-15-finding_HFRadar_currents.ipynb
-          - file: content/Code Gallery/data_access/2018-03-01-erddapy.ipynb
+          - file: content/Code Gallery/data_access_notebooks/2016-10-12-fetching_data.ipynb
```
Comment (ocefpaf) [inline jupyterbook/_toc.yml:9]: If we are going to use the directory to create the TOC file automatically we need to keep them, right?
Unless you have some a different idea.

File: jupyterbook/_config.yml [hunk 20]
```diff
@@ -0,0 +1,81 @@
...
+# Launch button settings
+launch_buttons:
+  notebook_interface        : jupyterlab  # The interface interactive links will activate ["classic", "jupyterlab"]
+  binderhub_url             : https://binder.pangeo.io  # The URL of the BinderHub (e.g., https://mybinder.org)
+  jupyterhub_url            : ""  # The URL of the JupyterHub (e.g., https://datahub.berkeley.edu)
+  thebe                     : false  # Add a thebe button to pages (requires the repository to run on Binder)
+  colab_url                 : " " # The URL of Google Colab (https://colab.research.google.com)
+
+repository:
+  url                       : https://github.com/lohithmunakala/test_jupyterbook # The URL to your book's repository
```
Comment (ocefpaf) [inline jupyterbook/_config.yml:72]: I'm guessing this is for testing, right? Let's not forget to change it before merging!

Comment (lohithmunakala) [reply jupyterbook/_config.yml:72]: Forgot to make this change. Will change and commit.

File: jupyterbook/_toc.yml (hunk 19 above)
Comment (lohithmunakala) [reply jupyterbook/_toc.yml:9]: Yes. I think we do. Will have to check though while writing the script.

**Comment by lohithmunakala** [comment]
@ocefpaf, there is a new format for the `_toc.yml` file in the latest update (0.11.1) for jupyter{book}. I have changed the _toc.yml according to that. https://github.com/ioos/notebooks_demos/pull/409/commits/d84e8699aab2ccb2c683a08e4d0d9367ce9c8766

File: .github/workflows/deploy-docs.yml [hunk 21]
```diff
@@ -23,6 +23,8 @@ jobs:
          conda create --name IOOS --file .binder/conda-linux-64.lock
          source activate IOOS
          conda install doctr
+      ## Adding install jupyter{book} command
```
Comment (ocefpaf) [inline .github/workflows/deploy-docs.yml:26]: This is REALLY a minor issue. But I kind of like the comment `#` to be aligned with the code. We can fix that later though.

File: .github/workflows/deploy-docs.yml [hunk 22]
```diff
@@ -23,6 +23,8 @@ jobs:
          conda create --name IOOS --file .binder/conda-linux-64.lock
          source activate IOOS
          conda install doctr
```
Comment (ocefpaf) [inline .github/workflows/deploy-docs.yml:25]: We can remove this line. We are no longer using `doctr` to publish to the page.

File: jupyterbook/_config.yml (hunk 16 above)
Comment (ocefpaf) [reply jupyterbook/_config.yml:48]: @lohithmunakala do you mind capturing @MathewBiddle's comment here in a new issue so we can address it later?

File: jupyterbook/content/Other Resources/ioos_installation_conda.md [hunk 23]
```diff
@@ -0,0 +1,129 @@
...
+While the full Anaconda distribution will also work,
+it's faster to install Miniconda,
+and you install only the packages you need.
+If for some reason you decide later that you want the full Anaconda distribution,
+you can install it by typing `conda install anaconda` using miniconda.
+
+## Install
+
+Download and install the appropriate Miniconda installer from [http://conda.pydata.org/miniconda.html](http://conda.pydata.org/miniconda.html).
+With Anaconda you can create environments that use any Python version (e.g. Python 2.7 or Python 3.6),
```
Comment (ocefpaf) [inline jupyterbook/content/Other Resources/ioos_installation_conda.md:14]: The 2.7 and 3.6 in the text show how dated these instructions are :-/
I'll update them ASAP!

**Review by ocefpaf — APPROVED** [review]

**Comment by ocefpaf** [comment]
@lohithmunakala this may look like the "easy" part but I guess you saw that the devil is in the details! You did an awesome work and hopefully the next PRs will be more fun coding rather than infrastructure discussion. Congrats on your first GSoC PR merged!

File: jupyterbook/_config.yml (hunk 16 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml:48]: Okay

**Comment by lohithmunakala** [comment]
> @lohithmunakala this may look like the "easy" part but I guess you saw that the devil is in the details! You did an awesome work and hopefully the next PRs will be more fun coding rather than infrastructure discussion. Congrats on your first GSoC PR merged!

@ocefpaf, @MathewBiddle Thanks to you guys!
